lib/policy_machine_storage_adapters/active_record/postgresql.rb

```diff
@@ -2,6 +2,95 @@
 
 module PolicyMachineStorageAdapter
   class ActiveRecord
+    class PolicyElementAssociation
+      def self.all_accessible_objects(associations, root_id: nil)
+        query = <<-SQL
+          id IN (
+            WITH RECURSIVE ancestors AS (
+            (
+              SELECT
+                objects.id AS parent_id
+                ,objects.id AS child_id
```

Comment:
why are the commas at the start of lines?

Reply:
i do this when writing queries because it's easier for me to tell when i'm missing a comma when adding/removing columns. it makes a big difference for my sql development speed, weirdly enough, but it's just a development quirk that i forgot to fix before i submitted the PR. sadly, no one else likes it but me 😢

Reply:
okay that's actually really very clever. i like it now